dbt/include/redshift/macros/relations/materialized_view/describe.sql

```diff
@@ -24,6 +24,22 @@
     {%- endset %}
     {% set _materialized_view = run_query(_materialized_view_sql) %}
 
+    {%- set _column_descriptor_sql -%}
+        SELECT
+            n.nspname AS schema,
+            c.relname AS table,
+            a.attname as column,
+            a.attisdistkey as is_dist_key,
+            a.attsortkeyord > 0 as is_sort_key
+        FROM pg_class c
+        JOIN pg_namespace n ON n.oid = c.relnamespace
+        JOIN pg_attribute a ON a.attrelid = c.oid
+        WHERE
+            n.nspname ilike '{{ relation.schema }}'
+            AND c.relname LIKE 'mv_tbl__{{ relation.identifier }}__%'
```

Comment:
Have you tried this with multiple config changes for the materialized view? Is there only ever one relation of the form `mv_tbl__{{ relation.identifier }}__%`? Do we need to look at the latest version? I'm asking because I'm genuinely not sure how Redshift manages that.

Reply:
For the way we're using it, they have only one relationship. 
However, I'll do a deeper analysis regarding that and get back to you with feedback. 
Good observation, thanks